build.sc

```diff
@@ -294,44 +291,33 @@ trait Core extends common.CoreModule with CrossSbtModule with ScalafmtModule {
     os.write(outputFile, contents, createFolders = true)
     PathRef(T.dest)
   }
-
   override def generatedSources = T {
     super.generatedSources() :+ buildInfo()
   }
 }
 
 object plugin extends Cross[Plugin](v.pluginScalaCrossVersions)
-
-trait Plugin extends common.PluginModule with CrossSbtModule with ScalafmtModule with ChiselPublishModule {
+trait Plugin extends CrossSbtModule with ScalafmtModule with ChiselPublishModule {
   override def artifactName = "chisel-plugin"
 
   // The plugin is compiled for every minor Scala version
   override def crossFullScalaVersion = true
 
   def millSourcePath = super.millSourcePath / os.up / "plugin"
-
   def scalaLibraryIvy = v.scalaLibrary(crossScalaVersion)
-
   def scalaReflectIvy = v.scalaReflect(crossScalaVersion)
-
   def scalaCompilerIvy: Dep = v.scalaCompiler(crossScalaVersion)
+
+  def ivyDeps = super.ivyDeps() ++ Agg(scalaLibraryIvy, scalaReflectIvy, scalaCompilerIvy)
 }
 
 object chisel extends Cross[Chisel](v.scalaCrossVersions)
-
-trait Chisel extends common.ChiselModule with CrossSbtModule with ScalafmtModule {
+trait Chisel extends CrossSbtModule with HasScala2MacroAnno with ScalafmtModule {
```

Comment:
> any ideas on why the integration tests are failing to get the plugin in their build unit?

@adkian-sifive `Chisel` also needs the compiler plugin `HasScala2Plugin`, that's why integration tests are failing. (regular CI is failing for the same reason)

Reply:
yeah that's it, thanks